Makefile

```diff
@@ -34,7 +34,18 @@ dev_up: ## Start development environment
 
 .PHONY: dev
 dev: ## Attach to the development environment if running
-	aws-vault exec "${AWS_PROFILE}" -- docker-compose -f docker-compose.dev.yml exec dev /bin/bash
+	@aws-vault exec "${AWS_PROFILE}" -- \
+	docker-compose -f docker-compose.dev.yml exec \
+		-e AWS_ACCESS_KEY_ID \
+		-e AWS_ACCOUNT_ID \
+		-e AWS_DEFAULT_REGION \
+		-e AWS_REGION \
+		-e AWS_SDK_LOAD_CONFIG \
+		-e AWS_SECRET_ACCESS_KEY \
+		-e AWS_SECURITY_TOKEN \
+		-e AWS_SESSION_EXPIRATION \
+		-e AWS_SESSION_TOKEN \
+		dev /bin/bash
```

Comment:
@mr337 - you were correct the other day that to re-inject AWS values I have to specify them explicitly. Ran into this today while trying to write to `chamber` after my session logged out. Fortunately with this change you just exit the container and re-enter and you're set!